web/telemetry/host_telemetry.go

```diff
@@ -0,0 +1,61 @@
+package telemetry
+
+import (
+	"errors"
+	"time"
+
+	"github.com/trento-project/trento/web/models"
+	"gorm.io/gorm"
+)
+
+type HostTelemetry struct {
+	InstallationID string    `json:"installation_id"`
```

Comment:
I would say that `InstallationID` and `AgentID` should go outside of this method, being them installation and agent generic. Otherwise, we will need to replicate them all over the telemetries

Reply:
This is what gets published to the telemetry collector, every "point" of a measurement has both installation id and agent id since it is atomic and it represents the telemetry of a single host in that precise moment. I wouldn't move this outside.

Reply:
Yup, this is going to be followed up, thanks.